src/raspberry_pi_motion_sensor/tests/test_motion_interface.py

```diff
@@ -0,0 +1,49 @@
+import sys
+from multiprocessing import Queue
+
+import fake_rpi
+
+from .. import motion_interface
+
+sys.modules["RPi"] = fake_rpi.RPi
+sys.modules["RPi.GPIO"] = fake_rpi.RPi.GPIO
```

Comment:
Please put `sys.modules["RPi"] = fake_rpi.RPi` and `sys.modules["RPi.GPIO"] = fake_rpi.RPi.GPIO` before `from .. import motion_interface` as in the past. In the most updated master branch, `isort` will ignore `test_motion_interface.py` so you can order these statements correctly to satisfy both `isort` and `pytest`